understanding/use-of-color.html

```diff
@@ -77,7 +100,7 @@ <h2>事例</h2>
                <li><p><strong>必須項目を示すために色とテキストを用いている入力フォーム</strong></p>
                   
                   
-                  <p>ある入力フォームには、必須項目と任意項目の両方がある。入力フォームの先頭にある説明文には、必須項目のラベルが赤字になっていて、「必須」というテキストによる代替のあるアイコンも付いている。赤字とアイコンの両方が、フォームのテキストフィールドとプログラム的に関連付けられているので、支援技術の利用者は必須項目を判断できる。</p>
+                  <p>ある入力フォームには、必須項目と任意項目の両方がある。入力フォームの先頭にある説明文では、必須項目のラベルが赤字になっていてアイコンも付いていることが説明されている。任意フィールドのラベルと必須フィールドの赤いラベルの違いを知覚できない利用者でも、赤いラベルに隣接するアイコンで視認することができる。</p>
```

Comment:
topは先頭にはならないのでは…2文目は意訳っぽくしてみました。
fieldsは項目で統一してみました。

```suggestion
                  <p>ある入力フォームには、必須項目と任意項目の両方がある。入力フォームの上部にある説明文では、必須項目は赤字とアイコンで説明されている。任意項目のラベルと必須フィールドの赤いラベルの違いを知覚できない利用者でも、赤いラベルに隣接するアイコンで視認することができる。</p>
```

Reply:
2文目の意訳、異論ありません。fields に対する訳語の揺れも、ご指摘ありがとうございます。